packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -303,46 +312,7 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
             <AngleLeftIcon />
           </button>
           <ul className={css(styles.tabsList)} ref={this.tabList} onScroll={this.handleScrollButtons}>
-            {filteredChildren.map((child, index) => {
-              const {
-                title,
-                eventKey,
-                tabContentRef,
-                id: childId,
-                tabContentId,
-                className: childClassName = '',
-                ouiaId: childOuiaId,
-                // eslint-disable-next-line @typescript-eslint/no-unused-vars
-                isHidden,
-                ...rest
-              } = child.props;
-              let ariaControls = tabContentId ? `${tabContentId}` : `pf-tab-section-${eventKey}-${childId || uniqueId}`;
-              if ((mountOnEnter || unmountOnExit) && eventKey !== localActiveKey) {
-                ariaControls = undefined;
-              }
-              return (
-                <li
-                  key={index}
-                  className={css(
-                    styles.tabsItem,
-                    eventKey === localActiveKey && styles.modifiers.current,
-                    childClassName
-                  )}
-                >
-                  <TabButton
-                    className={css(styles.tabsLink)}
-                    onClick={(event: any) => this.handleTabClick(event, eventKey, tabContentRef, mountOnEnter)}
-                    id={`pf-tab-${eventKey}-${childId || uniqueId}`}
-                    aria-controls={ariaControls}
-                    tabContentRef={tabContentRef}
-                    ouiaId={childOuiaId}
-                    {...rest}
-                  >
-                    {title}
-                  </TabButton>
-                </li>
-              );
-            })}
+            {filteredChildren}
```

Comment:
I'm a little confused why this mapping is no longer needed, did the structure of `filteredChildren` change?

Reply:
The tab component used to return null, i.e. render nothing. So filtered children was just an array of objects. I needed a ref on the tab component for tooltips, so I had to forward ref and render an instance to attach a ref to.